libs/wingsdk/src/util/platform-specific.ts

```diff
@@ -83,7 +88,44 @@ const getPlatformSpecificValue = (
 const getPlatformSpecificValuesFromFile = (path: string, file: string) => {
   const data = readFileSync(file);
   const yamlObj = parse(data.toString());
-  return yamlObj[`${path}`];
+  const index = getPathFromFile(path, yamlObj);
+  return yamlObj[`${index}`];
+};
+
+const checkPathFromCommandLine = (
+  objPath: string,
+  cmdPath: string
+): boolean => {
+  if (objPath === cmdPath) {
+    return true;
+  }
+  // check if it's a test
```

Comment:
not sure If I understand but cant we just check the process.env.WING_IS_TEST

Reply:
@marciocadev do you need any assistance here?